gnomad/utils/filtering.py

```diff
@@ -341,3 +341,17 @@ def filter_to_clinvar_pathogenic(
         t.count_rows() if isinstance(t, hl.MatrixTable) else t.count(),
     )
     return t
+
+
+def remove_fields_from_globals(global_field: List[str], fields_to_remove: List[str]):
```

Comment:
now that this function has been moved to the common repo/taken out of the context of VCF export, I wonder if there's a better name? I actually forgot what this did and thought this function was going to remove globals from HTs/MTs. maybe `remove_fields_from_constants` is better?